central/views/imagecve/view_impl.go

```diff
@@ -190,13 +191,46 @@ func (v *imageCVECoreViewImpl) GetImageIDs(ctx context.Context, q *v1.Query) ([]
 }
 
 func withSelectCVEIdentifiersQuery(q *v1.Query) *v1.Query {
+	// For pagination and sort to work properly, the filter query to get the CVEs needs to
+	// include the fields we are sorting on.  At this time custom code is required when
+	// sorting on custom sort fields.  For instance counts on the Severity column based on
+	// a value of that column
```

Comment:
Is a custom sort field an aggregate?

Reply:
yeah, an aggregate with a filter on a field.  So all of the ones listed here are aggregate counts on the same field.